examples/gallery/symbols/custom_symbols.py

```diff
@@ -4,10 +4,21 @@
 
 The :meth:`pygmt.Figure.plot` method can plot individual custom symbols
 by passing the corresponding symbol name together with the **k** shortcut to
-the ``style`` parameter. In total 41 custom symbols are already included of
-which the following plot shows five exemplary ones. The symbols are shown
-underneath their corresponding names. For the remaining symbols see the GMT
-Technical Reference :gmt-docs:`reference/custom-symbols.html`.
+the ``style`` parameter.
+
+In total 41 custom symbols are already included of which the following plot
+shows five exemplary ones. The symbols are shown underneath their
+corresponding names. For the remaining symbols see the GMT Technical
+Reference :gmt-docs:`reference/custom-symbols.html`.
+
+Beside these built-in custom symbols GMT allows users to define their own
+custom symbols. For this a specific macro language is used. An detailed
```

Comment:
```suggestion
custom symbols. For this a specific macro language is used. A detailed
```

Reply:
Included in https://github.com/GenericMappingTools/pygmt/pull/3186/commits/3638881fc34bfe785df0921e201cbb93a470e07d.